### PR TITLE
fix: synchronizing writes to localcommand file

### PR DESCRIPTION
**Pull request by patrickstuedi**
Attempt to fix race condition when writing local command file: https://confluentinc.atlassian.net/browse/KQE-532

### CODE REVIEW

**Review by agavra — APPROVED** [review]
thanks!

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/LocalCommandsFile.java [hunk 1]
```diff
@@ -59,7 +59,7 @@ public static LocalCommandsFile createWriteable(final File file) {
     return new LocalCommandsFile(file, true);
   }
 
-  public void write(final LocalCommand localCommand) throws IOException {
+  public synchronized void write(final LocalCommand localCommand) throws IOException {
```
Comment (AlanConfluent) [inline ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/LocalCommandsFile.java:62]: Do we want to do this on close as well, to ensure that we don't close it partway through writing a record?

**Review by AlanConfluent — APPROVED** [review]
Nice!  Looks good

File: ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/LocalCommandsFile.java (hunk 1 above)
Comment (patrickstuedi) [reply ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/LocalCommandsFile.java:62]: Good point.

**Pull request merged**